crates/biome_js_formatter/tests/specs/prettier/typescript/arrow/issue-2736.tsx

```diff
@@ -0,0 +1,19 @@
+type MenuDividerProps = {};
```

Comment:
Could you please relocate the test to the `crates/biome_js_formatter/tests/specs/js` directory? The tests in the `specs/prettier` directory are synchronized from the Prettier repository. It is not recommended to add tests in that directory

Reply:
I'll do it first thing tomorrow morning.